lib/optimizely/helpers/validator.rb

```diff
@@ -153,9 +153,11 @@ def boolean?(value)
       def same_types?(value_1, value_2)
         # Returns true if given values are of same types.
         #         false otherwise.
+        # Numeric values are considered as same type.
+
+        return true if value_1.is_a?(Numeric) && value_2.is_a?(Numeric)
 
         return true if boolean?(value_1) && boolean?(value_2)
-        return true if value_1.is_a?(Integer) && value_2.is_a?(Integer)
```

Comment:
Thanks for changing this! I just did some final QA for the 3.0 release, and I'm sure this would have been a problem if we hadn't changed it. We do expect numbers and floats to be interchangeable for the purposes of audience targeting. 👌